verifier/gaiax.go

```diff
@@ -44,17 +45,25 @@ func InitGaiaXRegistryVerificationService(verifierConfig *configModel.Verifier)
 	return verifier
 }
 
-func (v *GaiaXRegistryVerificationService) VerifyVC(verifiableCredential VerifiableCredential, verificationContext VerificationContext) (result bool, err error) {
-	if v.validateAll || slices.Contains(v.credentialTypesToValidate, verifiableCredential.GetCredentialType()) {
+func (v *GaiaXRegistryValidationService) ValidateVC(verifiableCredential *verifiable.Credential, verificationContext ValidationContext) (result bool, err error) {
```

Comment:
can we remove/rename verificationContext?

Reply:
done